### PR TITLE
fix input blur issue, update build

### PR DESCRIPTION
**Pull request by kltan**
/cc @ryan-roemer @geekdave

The library by default wraps <span class="twitter-typeahead"></span>
around the input field, internally that trigger some sort of append
where input element is taken out of the dom tree and re-appended.
This caused some problems with mobile browser, as you see when
an input field is taken out from the dom tree it loses focus.
Customers are enraged when their half entered field suddenly loses focus.
Another thing is that you cannot use $input.click() or $input.focus()
to trigger focus and keyboard on the field in mobile browser.
The solution is to wrap <span class="twitter-typeahead"></span> in server side html.

### CODE REVIEW

File: dist/bloodhound.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*!
  * typeahead.js 0.10.5
  * https://github.com/twitter/typeahead.js
- * Copyright 2013-2014 Twitter, Inc. and other contributors; Licensed MIT
+ * Copyright 2013-2015 Twitter, Inc. and other contributors; Licensed MIT
```
Comment (kltan) [inline dist/bloodhound.js:4]: auto change by grunt build script

**Comment by kltan** [comment]
`npm test` passed

File: src/typeahead/typeahead.js [hunk 2]
```diff
@@ -351,9 +351,25 @@ var Typeahead = (function() {
...
+    // where input element is taken out of the dom tree and re-appended.
+    // This caused some problems with mobile browser, as you see when
+    // an input field is taken out from the dom tree it loses focus.
+    // Customers are enraged when their half entered field suddenly loses focus.
+    // Another thing is that you cannot use $input.click() or $input.focus()
+    // to trigger focus and keyboard on the field in mobile browser.
+    // The solution is to wrap <span class="twitter-typeahead"></span> in server side html.
+    // In order to maximize compatibility with whomever is using this library
+    // I opted to do this detection below
+    if (!$input.parent().hasClass("twitter-typeahead")) {
```
Comment (ryan-roemer) [inline src/typeahead/typeahead.js]: GLOBAL: Please match quotation style of upstream repo. Here, that means single quotes.

**Comment by ryan-roemer** [comment]
@kltan -- Please fix quotes, rebuild, re-test, and rest Meta JS LGTM.

In the future, please open branches from this repo for PRs, so we all can easily jump in.

/cc @geekdave

File: src/typeahead/typeahead.js (hunk 2 above)
Comment (kltan) [inline src/typeahead/typeahead.js]: Ah missed that. Muscle memory.

**Comment by ryan-roemer** [comment]
@kltan -- Can you please also open an upstream PR for real repository? Thanks!

**Pull request merged**